{"name": "Логин юзера с некорректным паролем или логином", "status": "passed", "description": "Проверяем, что при логине с некорректным паролем или логином возвращается ошибка", "steps": [{"name": "Извлекаем данные для авторизации юзера", "status": "passed", "parameters": [{"name": "credentials", "value": "{'email': 'kiselevterenti+67916022@example.com', 'password': 'aqcpewqzfb', 'name': 'arfagvhcet'}"}], "start": 1723315047307, "stop": 1723315047307}], "parameters": [{"name": "login_type", "value": "'password'"}], "start": 1723315047026, "stop": 1723315047606, "uuid": "ff90a368-9cc0-4317-9a4c-39646b1392be", "historyId": "278efd8893f3cbe3f9dce85f70829503", "testCaseId": "07f5661e1ab0cebc2b1da901e6822b05", "fullName": "tests.test_login_user.TestLoginUser#test_login_with_wrong_credentials", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "MacBook-Pro-4.local"}, {"name": "thread", "value": "11097-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}